naptime/src/main/scala/org/coursera/naptime/router2/MacroImpls.scala

```diff
@@ -460,14 +461,42 @@ class MacroImpls(val c: blackbox.Context) {
             org.coursera.courier.templates.DataTemplates.DataConversion.SetReadOnly)
         """
       }
-      // TODO: handle input, custom output bodies, and attributes
+
+      val vagueTypes = List("Unit", "play.api.mvc.AnyContent", "Any")
```

Comment:
do you mind adding a comment what `vague` means here? is it just that we'd rather not return `Some[...]` for the input body/custom output bodies for these types, which are 'defaults' if not user specified?

Reply:
👍 thanks for the request, added.